### DISCUSSION TITLE
Module 5.1 error

### DISCUSSION THREAD

**Discussion by EdSpiezio-Runyon** [ua-gist604b-f24/syllabus#15] [General]
Has anyone else tried to launch the codespace in module 5.1? I keep getting the following error and can't seem to get a successful rebuild.

This codespace is currently running in recovery mode due to a configuration error. Please review the creation logs, update your dev container configuration as needed, and run the "Rebuild Container" command to rectify.

**Comment by aaryno** [comment]
Use an 8-core machine type for your codespace.

**Reply by travisz09** [reply]
I was able to load codespaces with an 8-core machine just fine. I believe there was an update earlier today. If you accepted the assignment earlier, you may have to try accepting it again.

I won't be able to finish tonight, but I have gotten through step 8 without issue.

**Reply by EdSpiezio-Runyon** [reply]
Something got it working. It worked fine for me this time.

**Comment by travisz09** [comment]
Also, this is unrelated to @EdSpiezio-Runyon's issue, but there is a small logical error in part of the assignment, here:

![Screenshot 2024-11-11 202258](https://github.com/user-attachments/assets/e2f454f7-61a0-4da1-814e-a108b809e86f)

The source file did not get updated for the table nyc_homicides. The referenced nyc_census_blocks.shp source file was loaded in a prior step, and is subsequently duplicated in the following code block.

I do not know if this will be a source of any errors in the questions or tutorials, or if it is just benign, but the correct line of code (I believe) should be;
`shp2pgsql -s 26918 -c -g geom data/nyc_homicides.shp public.nyc_homicides | psql -d nyc`
`...`

**Reply by EdSpiezio-Runyon** [reply]
Good catch!
